### PR TITLE
common: roadmap updated after demo on 9/17/2020

### PR DESCRIPTION
**Pull request by grom72**
Roadmap before last sprint of release 0.9

---
This change is [Reviewable](https://reviewable.io/reviews/pmem/rpma/416)

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
-# **The librpma roadmap [09/04/2020]**
-
-This document presents the librpma library roadmap as it shaped on 09/04/2020. It lists high-level features that are going to be released with the library release 0.9 and features that are planned to be developed later.
+# **The librpma roadmap [21/04/2020]**
+This document presents the librpma library roadmap as it shaped on 21/04/2020. It lists high-level features that are going to be released with the library release 0.9 and features that are planned to be developed later.
```
Comment (pbalcer) [inline ROADMAP.md:2]: `the current state of the library as of X`.

**Review by ldorau — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/416)* status: all files reviewed, 1 unresolved discussion (waiting on @pbalcer)

___
*[ROADMAP.md, line 2 at r1](https://reviewable.io/reviews/pmem/rpma/416#-MHjebF5Hk5h_oYGgVXx-r1-2:-MHkEu-6-e5AOaIyqDJP:b-8fx2u7) ([raw file](https://github.com/pmem/rpma/blob/6cbde149708051573902cb68f8b44ce74a89a5e1/ROADMAP.md#L2)):*
<details><summary><i>Previously, pbalcer (Piotr Balcer) wrote…</i></summary><blockquote>

`the current state of the library as of X`.
</blockquote></details>

:+1:

**Review by grom72 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/416)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @ldorau and @pbalcer)

___
*[ROADMAP.md, line 2 at r1](https://reviewable.io/reviews/pmem/rpma/416#-MHjebF5Hk5h_oYGgVXx-r1-2:-MHoQMRLAXeBWkmuHIyz:b-896fix) ([raw file](https://github.com/pmem/rpma/blob/6cbde149708051573902cb68f8b44ce74a89a5e1/ROADMAP.md#L2)):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

:+1: 
</blockquote></details>

Done.

**Review by ldorau — APPROVED** [review]
> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/416)* status: all files reviewed, 1 unresolved discussion (waiting on @pbalcer)

**Review by ldorau — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/416#-:-MHp143h3YS7Bobgji61:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @pbalcer)

**Review by ldorau — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/416)* status: all files reviewed, 2 unresolved discussions (waiting on @grom72 and @pbalcer)

___
*[ROADMAP.md, line 30 at r2](https://reviewable.io/reviews/pmem/rpma/416#-MHp2NlR8f84toS9dBFJ:-MHp2NlR8f84toS9dBFK:b-qkojkk) ([raw file](https://github.com/pmem/rpma/blob/1216fbc10457cd3d2f746278ee1fead257ba9da7/ROADMAP.md#L30)):*
> ```Markdown
> :ballot_box_with_check:|Peer configuration|Peer configuration support|09/18/2020
> :ballot_box_with_check:|Legacy OS support|Build system support for legacy Linux distributions|09/18/2020
> :ballot_box_with_check:|GPSPM example|An example that demonstrates how to implement GPSPM Flushing method on the top of the librpma library|09/18/2020
> ```

flushing (small letter, not capital)

**Review by ldorau — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/416)* status: all files reviewed, 1 unresolved discussion (waiting on @pbalcer)

___
*[ROADMAP.md, line 30 at r2](https://reviewable.io/reviews/pmem/rpma/416#-MHp2NlR8f84toS9dBFJ:-MHp2jGt-DyPmMrCqRmo:b4rr5kv) ([raw file](https://github.com/pmem/rpma/blob/1216fbc10457cd3d2f746278ee1fead257ba9da7/ROADMAP.md#L30)):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

flushing (small letter, not capital)
</blockquote></details>

No, it is OK, because the rest are also in capital letters.

**Review by ldorau — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/416)* status: all files reviewed, 3 unresolved discussions (waiting on @grom72 and @pbalcer)

___
*[ROADMAP.md, line 14 at r2](https://reviewable.io/reviews/pmem/rpma/416#-MHp39PkByuKLFN8bsOR:-MHp39PkByuKLFN8bsOS:b-tponzw) ([raw file](https://github.com/pmem/rpma/blob/1216fbc10457cd3d2f746278ee1fead257ba9da7/ROADMAP.md#L14)):*
> ```
> RDMA.Read
> ```

Why is there a dot between? You sometimes write with dot between and sometimes not. 
Is it needed? Maybe remove all dots?
___
*[ROADMAP.md, line 20 at r2](https://reviewable.io/reviews/pmem/rpma/416#-MHp3dze1YbJQ7r_z9QL:-MHp3dze1YbJQ7r_z9QM:b-41116u) ([raw file](https://github.com/pmem/rpma/blob/1216fbc10457cd3d2f746278ee1fead257ba9da7/ROADMAP.md#L20)):*
> ```
> RDMA Write
> ```

No dot here

**Review by ldorau — COMMENTED** [review]
Link to see it:
https://github.com/grom72/rpma/blob/roadmap/ROADMAP.md

> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/416#-:-MHp3rQt3iNmdE2w9F_n:b-htu2gy)* status: all files reviewed, 3 unresolved discussions (waiting on @grom72 and @pbalcer)

**Review by ldorau — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/416)* status: 0 of 1 files reviewed, 4 unresolved discussions (waiting on @grom72, @ldorau, and @pbalcer)

___
*[ROADMAP.md, line 14 at r3](https://reviewable.io/reviews/pmem/rpma/416#-MHpHD3W7StNiZDDf3Y3:-MHpHD3XBPAGV02vzTDI:b-emye5i) ([raw file](https://github.com/pmem/rpma/blob/801270f002f56476f57cfe34b4bb68d86e7e45da/ROADMAP.md#L14)):*
> ```
> *RDMARead*
> ```

No space between

**Review by ldorau — COMMENTED** [review]
> Reviewed 1 of 1 files at r3.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/416)* status: all files reviewed, 4 unresolved discussions (waiting on @grom72 and @pbalcer)

**Review by grom72 — COMMENTED** [review]
> Reviewed 1 of 1 files at r4.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/416)* status: all files reviewed, 4 unresolved discussions (waiting on @ldorau and @pbalcer)

___
*[ROADMAP.md, line 14 at r2](https://reviewable.io/reviews/pmem/rpma/416#-MHp39PkByuKLFN8bsOR:-MHp_tpUBU_svrrevc0Y:b-896fix) ([raw file](https://github.com/pmem/rpma/blob/1216fbc10457cd3d2f746278ee1fead257ba9da7/ROADMAP.md#L14)):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

> ```
> RDMA.Read
> ```

Why is there a dot between? You sometimes write with dot between and sometimes not. 
Is it needed? Maybe remove all dots?
</blockquote></details>

Done.
___
*[ROADMAP.md, line 20 at r2](https://reviewable.io/reviews/pmem/rpma/416#-MHp3dze1YbJQ7r_z9QL:-MHp_uySENJJKRym0uLt:b-896fix) ([raw file](https://github.com/pmem/rpma/blob/1216fbc10457cd3d2f746278ee1fead257ba9da7/ROADMAP.md#L20)):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

> ```
> RDMA Write
> ```

No dot here
</blockquote></details>

Done.
___
*[ROADMAP.md, line 30 at r2](https://reviewable.io/reviews/pmem/rpma/416#-MHp2NlR8f84toS9dBFJ:-MHpa1qY6uSFprcSc4Lm:b-tvl0bc) ([raw file](https://github.com/pmem/rpma/blob/1216fbc10457cd3d2f746278ee1fead257ba9da7/ROADMAP.md#L30)):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

No, it is OK, because the rest are also in capital letters.
</blockquote></details>

I feel lost
___
*[ROADMAP.md, line 14 at r3](https://reviewable.io/reviews/pmem/rpma/416#-MHpHD3W7StNiZDDf3Y3:-MHpaMZQ9_rFYqIsmbsK:b-896fix) ([raw file](https://github.com/pmem/rpma/blob/801270f002f56476f57cfe34b4bb68d86e7e45da/ROADMAP.md#L14)):*
<details><summary><i>Previously, ldorau (Lukasz Dorau) wrote…</i></summary><blockquote>

> ```
> *RDMARead*
> ```

No space between
</blockquote></details>

Done.

**Review by ldorau — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/416)* status: all files reviewed, 1 unresolved discussion (waiting on @pbalcer)

**Review by ldorau — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/416#-:-MHpcxuE7vTeLcxiJeFF:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @pbalcer)

**Review by ldorau — APPROVED** [review]
> Reviewed 1 of 1 files at r4.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/416)* status: all files reviewed, 1 unresolved discussion (waiting on @pbalcer)

**Review by ldorau — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/416)* status: all files reviewed, 1 unresolved discussion (waiting on @pbalcer)

___
*[ROADMAP.md, line 2 at r1](https://reviewable.io/reviews/pmem/rpma/416#-MHjebF5Hk5h_oYGgVXx-r1-2:-MHwBo5-0QriugYcH8hd:bjqjnzy) ([raw file](https://github.com/pmem/rpma/blob/6cbde149708051573902cb68f8b44ce74a89a5e1/ROADMAP.md#L2)):*
<details><summary><i>Previously, grom72 (Tomasz Gromadzki) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

@pbalcer it looks like fixed ;-) merging ...

**Pull request merged**